README.md

```diff
@@ -80,6 +80,18 @@ To run tests
  $ npm test -- -u --coverage
 ```
 
+### Creating a Sass build
+```
+- To compile Sass to CSS each time you make a change to a file with a .scss file 
```

Comment:
Should it be Sass or SASS?

Reply:
According to [this source](https://sass-lang.com/), they use `Sass` and not `SASS`.

Reply:
Then in that case, any reason why line 90 below uses SASS?

Reply:
Thanks for the keep eye @dkayiwa, let me fix it in a moment.